service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -78,4 +78,7 @@
     @NotNull(message = "You need to add details to children")
     @Valid
     private final List<@NotNull(message = "You need to add details to children")Element<Child>> children1;
```

Comment:
Does Sam's changes not remove children 1 is replace for children?

Reply:
Children1 is current model.